### PR TITLE
CHE-243: Return link header describing page links

### PR DESCRIPTION
**Pull request by evoevodin**
Add page links for gettings users.
@skabashnyuk please review

### CODE REVIEW

File: core/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/dao/ldap/AdminUserDaoImpl.java [hunk 1]
```diff
@@ -64,19 +74,62 @@ public AdminUserDaoImpl(AccountDao accountDao,
...
+
+    // TODO: consider WeakReference holder for returned value.
+    // as this operation may be expensive on large medium/large data sets
+    private int getTotalUsersCount() throws ServerException {
+        InitialLdapContext context = null;
+        try {
+            context = contextFactory.createContext();
+            final SearchControls controls = new SearchControls();
+            controls.setSearchScope(SearchControls.SUBTREE_SCOPE);
+            controls.setReturningAttributes(EMPTY_ARRAY);
```
Comment (skabashnyuk) [inline core/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/dao/ldap/AdminUserDaoImpl.java]: is this call mandatory?

Comment (evoevodin) [inline core/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/dao/ldap/AdminUserDaoImpl.java]: yes, there is no way to count elements using ldap filters, the only way is to count elements in enumeration, so it is much cheaper to return N matched elements which are empty and count them.

Comment (skabashnyuk) [inline core/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/dao/ldap/AdminUserDaoImpl.java]: When add plz this information to EMPTY_ARRAY constant.

Comment (evoevodin) [inline core/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/dao/ldap/AdminUserDaoImpl.java]: I will add this information to method javadoc, thank you

**Comment by skabashnyuk** [comment]
ок

**Pull request merged**